addons/l10n_in/models/account_invoice.py

```diff
@@ -22,7 +17,7 @@ def _compute_amount_total_words(self):
             ('overseas', 'Overseas'),
             ('special_economic_zone', 'Special Economic Zone'),
             ('deemed_export', 'Deemed Export')
-        ], string="GST Treatment", readonly=True, states={'draft': [('readonly', False)]})
+        ], string="GST Treatment", compute="_compute_l10n_in_gst_treatment", store=True, readonly=False)
```

Comment:
last thing: you need to adapt the view:
https://github.com/odoo/odoo/blob/a96d3a860d58b8d1987eab58977d21b28afbc900/addons/l10n_in/views/account_invoice_views.xml#L11
By adding: `'readonly': [('state', '!=', 'draft')]`

Reply:
Thanks :)